lib/cache.js

```diff
@@ -36,19 +36,18 @@ exports.update = (done) => {
       return done(err);
     }
     // Creating cache folder
-    fs.mkdirs(CACHE_FOLDER, (err) => {
-      if (err) {
+    fs.ensureDir(CACHE_FOLDER)
```

Comment:
Looks good. However, the parent function would need to be converted as I did. I'll handle that in my PR. If it gets too messy to look at, I'll use callbacks instead.